.github/workflows/static_checks.yml

```diff
@@ -6,9 +6,9 @@ on:
   workflow_dispatch:
   pull_request:
   push:
-    branches:    
+    branches:
       - develop
-      
+
```

Comment:
Undo these nit changes.

Reply:
Actually, these are unwanted extra spaces. So, we may want to keep it. Please confirm it again.

Reply:
Actually, these are unwanted extra spaces. So, we may want to keep it. Please confirm it again.

Reply:
Okay.